<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/addstu.png</normaloff>:/images/addstu.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	
	border-color: rgb(255, 255, 255);
	
	font: 10pt &quot;黑体&quot;;

	
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-color: rgb(176, 213, 223);
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_clear">
          <property name="text">
           <string>清空学生信息</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="9">
         <widget class="QTableWidget" name="tableWidget">
          <property name="styleSheet">
           <string notr="true">
QTableWidget::item{
	font:14px &quot;微软雅黑&quot;;
	color:#677483;
	border:none;
	text-align:center;
	
	
}

QHeaderView::section{
	color:gray;
	font-size:14px;
	font-weight:bold;
	text-align:center;
	height:60px;
	color:#909399;
	border:1px gray;
	
	
	

}</string>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="autoScrollMargin">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>全选</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_delete">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
	background-color:#f17666;
}</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_add">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
	background-color:#6FB7C9;
	
}</string>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/add.png</normaloff>images/add.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_stucout">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QPushButton" name="pushButton_search">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
	background-color:#93d5dc;
	
}</string>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/images/search1.png</normaloff>:/images/search1.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_simulation">
          <property name="text">
           <string>模拟按键</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLineEdit" name="lineEdit_search">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>33</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: 1px rgb(0, 0, 0);
font: 10pt &quot;黑体&quot;;
border-radius:5;</string>
          </property>
          <property name="placeholderText">
           <string>请输入需要搜索的名字</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_modify">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
	background-color:#93d5dc;
	
}</string>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>67</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>52</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>302</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:hover{
	background-color:#f17666;
}</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_image">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/images/users.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_admin">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Bahnschrift SemiBold SemiConden</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>用户名：admin</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>学生信息管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
